ui/src/containers/VisualiseResults/SourceResults.js

```diff
@@ -69,21 +76,43 @@ const formatKeyToFriendlyString = (key) => {
   return JSON.stringify(key, null, 2);
 };
 
-const getAxisLabel = (axis, visualisation, type) => {
-  if (type !== XVSY) {
-    return getAxesString(axis, visualisation, type, false);
+const getGroupAxisLabel = (visualisation) => {
+  switch (visualisation.get('type')) {
+    // Correlation Chart type
+    case XVSY:
+      return visualisation.getIn(['axesgroup', 'searchString']) || 'Group';
```

Comment:
I replaced `.getIn` second argument with `||` operator to show a default value, e.g. "Group" in this line, when "axesgroup.searchString" is empty string.